src/ceph-volume/ceph_volume/util/encryption.py

```diff
@@ -103,7 +103,10 @@ def dmcrypt_close(mapping):
         logger.debug('device mapper path does not exist %s' % mapping)
         logger.debug('will skip cryptsetup removal')
         return
-    process.run(['cryptsetup', 'remove', mapping])
+    try:
+        process.run(['cryptsetup', 'remove', mapping])
```

Comment:
The `process.run()` helper has a useful `stop_on_error` argument that defaults to `True` which causes the `RuntimeError` this is trying to avoid. So this could be simplified like:

```suggestion
       # don't be strict about the remove call, but still warn on the terminal if it fails
        process.run(['cryptsetup', 'remove', mapping], stop_on_error=False)
```

Reply:
done.
Thanks!